### DISCUSSION TITLE
Why is Smart Connections not opening?

### DISCUSSION THREAD

**Discussion by SSzretter** [brianpetro/obsidian-smart-connections#511] [Q&A] [answered]
In obsidian if I control-p and select view smart connections, nothing happens.

 The Smart Connections plugin accepted my api key and I have had prompts appear asking if I want to generate blocks which I did and can see that activity on openai and I see no errors in the debug console and can see the activity there as well.

When I choose view smart connections I see a message 'rendering result' in the debug console but nothing else.   

**The only thing out of the ordinary is when obsidian starts up I see an error in the debug console about a websocket connection failed to ws://localhost:3704  and a couple related errors after that.**

I am using OpenAI Text-3 Small , gpt-3.5-turbo.

**Comment by brianpetro** [comment, marked as answer]
Hi @SSzretter 

The views open in the right side panel. So make sure that the right side panel is visible. 

"Rendering result" in the log suggests the view is already open. 

And the websocket error is OK and won't impact anything.

Lastly, screenshots are always helpful so I can see what you're seeing. 

Let me know if that helps.

Thanks for trying Smart Connections 
🌴

**Reply by SSzretter** [reply]
I found it in the side panel, thank you!
